main.go

```diff
@@ -4,18 +4,25 @@ import (
 	"context"
 	"os"
 
+	"github.com/aws/aws-lambda-go/lambda"
+
+	"github.com/asecurityteam/nexpose-asset-attributor/pkg/assetattributor"
 	"github.com/asecurityteam/nexpose-asset-attributor/pkg/handlers/v1"
+	"github.com/asecurityteam/runhttp"
 	serverfull "github.com/asecurityteam/serverfull/pkg"
 	serverfulldomain "github.com/asecurityteam/serverfull/pkg/domain"
 	"github.com/asecurityteam/settings"
-	"github.com/aws/aws-lambda-go/lambda"
 )
 
 func main() {
 	ctx := context.Background()
-	greetingHandler := &v1.GreetingHandler{}
+	attributeHandler := &v1.AttributeHandler{
+		AssetAttributor: assetattributor.NewNoOpAssetAttributor(),
+		LogFn:           runhttp.LoggerFromContext,
```

Comment:
Did you mean for this to use `domain.LoggerFromContext`?

Reply:
yep, one sec. Thanks!